notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>What is the reason for using the CalibratedClassifierCV in addition to the SVM? You can still get probabilities from the svm using <code>model.predict_proba()</code></p><p> </p><p>It just seems like an extra step where its unclear its needed/ provides more accurate results than using svm directly.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>yes, we will add the graphs which explain the difference between how the probabilities are calculated with and without calibration</p>